src/pages/LinkList/components/LinksGrid/components/LinkRow/LinkRow.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Link } from "./../../interfaces/Link";
+import { FormattedMessage, useIntl, FormattedDate } from "react-intl";
+
+export interface LinkRowInterface {
+  filteredLinks: Link[];
+}
+const LinkRow: React.FC<LinkRowInterface> = ({ filteredLinks }) => {
+  const intl = useIntl();
+  return (
+    <>
+      {filteredLinks.length ? (
+        filteredLinks.map((el, index) => (
+          <tr key={index}>
+            <td>
+              <span> {el.link} </span>
+            </td>
+            <td>
+              <span className="dp-event-label">{el.status}</span>
+            </td>
+            <td>
+              <span>
+                {!el.verifiedAt || (
+                  <span title={intl.formatDate(el.verifiedAt)}>
+                    <FormattedDate value={el.verifiedAt} />
+                  </span>
+                )}
+              </span>
+            </td>
+          </tr>
+        ))
+      ) : (
+        <tr>
+          <td colSpan={3}>
```

Comment:
Just curiosity, why not use simple HTML here?

```suggestion
          <td colSpan="3">
```

Reply:
In my case, in this situation I try to use `"3"` instead of `{3}`. I leave the notation with braces for when it is a javascript expression, for example: `<td colSpan={obj.value}>`.

Reply:
It is not HTML actually, it is called JSX which is very similar to HTML

Reply:
> It is not HTML actually, it is called JSX which is very similar to HTML

Ok, but `<td colSpan="3">` does not work with JSX?

Reply:
I have tried it and it seems not

Reply:
You are right, JSX compiler understands that `"3"` is a string and TypeScript knows that `colSpan` requires a number, so you have to use `{3}`.

Probably, in Doppler WebApp @jcamposmk simply use `"3"` because they are using plain JavaScript in place of TypeScript.

<img width="1148" alt="image" src="https://user-images.githubusercontent.com/1157864/210276691-bd4ee39f-9d9d-42f8-8947-4a5a91c59273.png">

Thanks!